src/main/java/com/example/java/maven/cardGamePeterPan/Player.java

```diff
@@ -29,12 +29,11 @@ public int getHandSize() {
         return hand.size();
     }
 
-//    TODO what if player is a monkey?
     public void discardCardsFromHand(int firstCardIndex, int secondCardIndex) {
         validateCardIndex(firstCardIndex);
         validateCardIndex(secondCardIndex);
-        hand.remove(secondCardIndex);
-        hand.remove(firstCardIndex);
+        hand.remove(Math.max(firstCardIndex, secondCardIndex));
```

Comment:
GUT!

Reply:
but what if player will input same numbers? :D 
so it will be: `3` and `3` - IMHO this input should be also invalid, so one more validation before removal :D

Reply:
Done